### PR TITLE
Improve Windows Chrome upload help

### PR DESCRIPTION
**Pull request by HelenFoster**
The Windows half of https://github.com/Microsoft/pxt/issues/2465

Had to reorder some of the other instructions to have this make sense.

### CODE REVIEW

File: docs/device/usb.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ directly into the MICROBIT drive!
 
 ### ~
 
+### ~ hint
+
```
Comment (pelikhan) [inline docs/device/usb.md]: It sure we want this here. There are various different hints for each browser and it will overload this page.

**Comment by pelikhan** [comment]
Try to use the macro @boardname@ instead of micro:bit . Thanks for improving our docs!

File: docs/device/usb.md (hunk 1 above)
Comment (HelenFoster) [inline docs/device/usb.md]: Should we delete the other one as well then?

**Comment by HelenFoster** [comment]
> Try to use the macro @boardname@ instead of micro:bit

Mostly it's talking about the MICROBIT drive on the computer, so is there anything for that?

I didn't change much of the text in this document - it just looks like a lot of changes because it's in a different order.

**Comment by ganicke** [comment]
I suggest removing the hint(s) since the same info is in the browser specific pages. Try to connect the **Instructions** section to step (3) of "basic steps". I have a suggested replacement here: https://github.com/Microsoft/pxt-microbit/commit/20b385dbbe7b87408fa8531c146492e40a2041ab.

**Comment by HelenFoster** [comment]
I agree with @ganicke

**Pull request merged**